docs-content/getting-started/backend-logging/5_hosting/aws.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Amazon Web Services (AWS)
+slug: aws
+createdAt: 2022-03-28T20:31:15.000Z
+updatedAt: 2022-04-06T20:22:54.000Z
+---
+
+Deploying your application or infrastructure in AWS? Stream your logs to highlight to see everything in one place.
```

Comment:
Don't most AWS services support fluentd/firehose? Could we add something here that tells them that most services support it and link a few other examples? I think not mentioning anything else makes it sound like we only support ecs, when that's not the case?

Reply:
added!